ds_import/face.py

```diff
@@ -13,18 +13,20 @@ def __init__(self, vertices=None):
         """
         if vertices is not None:
             # validate vertices
-            if len(vertices) < 3:
-                raise ValueError("At least three vertices expected")
+            if len(vertices) == 3:
```

Comment:
Did you mean "if len(vertices) != 3" here?

Reply:
Yes. Will fix.